sample/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.3"
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    defaultConfig {
+        applicationId "org.wordpress.passcodelock.sample"
+        minSdkVersion 11
```

Comment:
The library declares `minSdkVersion 9`, but our implementation of `DefaultAppLock` is available on 14+. What if we bump the min SDK to 14 even in the library (and here in the sample app) ?

If not it's a viable solution, we should bump minsdk here to 14, otherwise the default passcode lock is not available at all. Also a comment in the readme of the library could help on this.